src/renderer.ts

```diff
@@ -1,13 +1,15 @@
 import * as nunjucks from 'nunjucks';
 import * as utils from './utils';
-import {Pod} from './pod';
+import Pod from './pod';
 import marked from 'marked';
 
 export class Renderer {
   pod: Pod;
+  kind: string;
```

Comment:
Using `kind` always felt kinda kludgy to me and the name was I think an artifact of a "Datastore kind". Instead of adding a new property can we instead do one of the following for instances where we need to test the type:

```
function Foo() {}
var foo = new Foo();

foo instanceof Foo;     // == true
foo.constructor.name;   // == "Foo"
Foo.name                // == "Foo"    
```

Reply:
Since we are going to be rewriting the render code over to template engines I'll leave this for now and when it gets rewritten we can test out using the `constructor.name`. I think it would work ok for the engines since there won't be very many different engines at a time.

Reply:
Actually, that is not a complicated change, I've removed it.